src/components/general/TextArea/styles.less

```diff
@@ -0,0 +1,145 @@
+@import '~styles/mixins.less';
+
+.textAreaContainer {
+   display: flex;
+    flex-direction: column;
+
+    .textAreaContent {
+        display: flex;
+        justify-content: space-between;
+        min-height: .grid-unit(15px)[];
```

Comment:
Is it possible to remove the min height here? 120px is pretty large. If it needs a min height maybe the size of 2 rows of text could be the minimum? Or maybe take the min height or styling as a prop so that the user of the component can specify the minimum height?

Reply:
I have now changed the min height to the size of two rows of textinput

Reply:
Great!